packages/core/src/components/navigation/Navbar/Navbar.tsx

```diff
@@ -77,6 +77,7 @@ function Navbar({
     NavbarRow,
     NavbarButtons,
     IconButton,
+    _experimentalSignInButton: SignInButton,
```

Comment:
nitpick: Can you rename it just to follow the previous pattern. I know we are using signInButton in the prop 🙈 

```suggestion
    _experimentalButtonSignIn: SignInButton,
```

Reply:
After that can you pls remove the unused component import in this file (line 14)

`import { ButtonSignIn } from 'src/components/ui/Button'`

Thank you!

Reply:
Done